### PR TITLE
Fix doc test

### PR DESCRIPTION
**Pull request by k-nasa**
- Fix lib.rs doc test
- Run ignored stream test
- Fix `stream::min` and `stream::max`

### CODE REVIEW

File: src/stream/stream/max.rs [hunk 1]
```diff
@@ -11,29 +10,23 @@ use crate::task::{Context, Poll};
 pin_project! {
     #[doc(hidden)]
     #[allow(missing_debug_implementations)]
-    pub struct MaxFuture<S, F, T> {
+    pub struct MaxFuture<S, T> {
```
Comment (dignifiedquire) [inline src/stream/stream/max.rs:13]: this seems unrelated, why this change?

Comment (k-nasa) [reply src/stream/stream/max.rs:13]: I think it is unnecessary type argument. 
Certainly that may be different from what  I'm doing with this PR.

**Comment by k-nasa** [comment]
If pass ci, I merge it.
